UnitsNet.OperatorOverloads/ArrayExtensions.cs

```diff
@@ -22,6 +22,21 @@ public static int[] ElementwiseSubtract(this int[] left, int[] right)
             return result;
         }
 
+        public static int[] ElementwiseAdd(this int[] left, int[] right)
```

Comment:
You can use LINQ's `.Zip()`, like so

```
	public static int[] ElementwiseAdd(this int[] left, int[] right)
	{
		return left.Zip(right, (l,r) => l + r).ToArray();
	}
```

However, it just picks the shortest length of the two, so you may want to keep the length check.